docs/topics/about-tools.adoc

```diff
@@ -10,6 +10,8 @@ The {ProductName} ({ProductShortName}) provides several tools to assist you in t
 
 * CLI
 * Web console
+* {ProductName} Operator
 * {PluginName} for Eclipse and Red Hat CodeReady Studio
-* {ProductShortName} extension for Visual Studio Code
+* {ProductShortName} extension for Visual Studio Code and Exlipse Che
```

Comment:
@RichardHoch "Eclipse" is not written correctly

Reply:
Fixed. \